README.md

```diff
@@ -1,71 +1,90 @@
-###Download the latest pre-built Windows Bridge for iOS SDK [here](https://aka.ms/winobjc_latest).
+# Welcome to the Windows Bridge for iOS Project
+The Windows Bridge for iOS (also referred to as WinObjC) is a Microsoft open-source project that provides an Objective-C development environment for Visual Studio and support for iOS APIs. The bridge allows you to create Universal Windows Platform (UWP) apps that will run on many Windows devices by re-using your Objective-C code and iOS APIs alongside Windows 10 features like Cortana and Live Tiles.
```

Comment:
r/Live Tiles/Windows Notifications

Overall, WNS is more easily used by the external developer. You pretty much need a WinPhone to see a live tile in action.